src/components/landing-layout/footer/version-info/version-info-link.tsx

```diff
@@ -18,9 +19,14 @@ export const VersionInfoLink: React.FC = () => {
 
   return (
     <Fragment>
-      <a {...cypressId('show-version-modal')} href={'#'} className={'text-light'} onClick={showModal}>
+      <Button
```

Comment:
What advantage does this have and why is this in this PR?

Reply:
Links to "#" are broken with the URL changes. Therefore I needed another element to trigger the version modal.